source/Conventions.rst

```diff
@@ -25,7 +25,7 @@ Navigation
 User Interface & Configuration
 ==============================
 
-.. _ui-cli
+.. _ui-cli:
 
 Command Line Interface (CLI)
 ++++++++++++++++++++++++++++
```

Comment:
ln 84-88: some overlap with Use-Protocol, especially for the status_message; maybe also explicitly link that page to here for more detail on implementation?